doc/Tex/RAM_SCB_viz.tex

```diff
@@ -1,12 +1,5 @@
-RAM-SCB output can be visualized in a number of different ways depending on
-the user's tastes and preferences.  However, a standard library for opening,
-manipulating, and plotting exists as a sub-module of Spacepy.  Users may
-find this freely available Python library at
-\href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively,
-development versions can be obtained via the project's
-\href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The
-RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
+RAM-SCB output can be visualized in a number of different ways depending on the user's tastes and preferences.  However, a standard library for opening, manipulating, and plotting exists as a sub-module of Spacepy.  Users may find this freely available Python library at \href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively, development versions can be obtained via the project's \href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
```

Comment:
spacepy.lanl.gov has been dead for a while, as has spacepy's sourceforge page.

These need updating to https://spacepy.github.io/ (docs), and https://github.com/spacepy/spacepy (repo).

Reply:
K